pyomo/contrib/sensitivity_toolbox/pynumero.py

```diff
@@ -62,7 +63,7 @@ def get_dsdp_dfdp(model, theta):
     Returns
     -------
     scipy.sparse.csc_matrix, csc_matrix, ComponentMap, ComponentMap
-        ds/dp (ns + np by np), df/dp (1 by np), row map, column map.
```

Comment:
Why did you change the dimension of ds/dp in the docstring without changing anything in the implementation? You just made this change (i.e. changing the dimension from `ns by np` to `ns+np by np`) in #3655

Reply:
This is a mistake.  Somehow, I accidently reverted to a older version of the file and things got a little mixed up.  I'll fix it.